src/ncp/spinel.h

```diff
@@ -409,6 +409,23 @@ typedef enum
      */
     SPINEL_PROP_JAM_DETECT_BUSY         = SPINEL_PROP_PHY_EXT__BEGIN + 4,
 
+    /// Jamming detection history bitmap (for debugging)
+    /** Format: `LL` (read-only)
+     *
+     * This value provides information about current state of jamming detection
+     * module for monitoring/debugging purpose. It returns a 64-bit value where
+     * each bit corresponds to one second interval starting with bit 0 for the
+     * most recent interval and bit 63 for the oldest intervals (63 sec earlier).
+     * The bit is set to 1 if the jamming detection module observed/detected
+     * high signal level during the corresponding one second interval.
+     *
+     * The value is read-only and is encoded as two uint32 values in
+     * little-endian format (first uint32 gives the lower bits corresponding to
+     * more recent history).
+     */
+    SPINEL_PROP_JAM_DETECT_HISTORY_BITMAP
```

Comment:
Added the new property description in `doc/spinel-protocol-src/spinel-feature-jam-detect.md` in the new commit push. 

Also ran the tool to update the `draft-spinel-protocol.html` and `draft-spinel-protocol.txt` but I'll push the changes to these in a separate commit/PR after this goes in master (to keep the diff on this commit smaller).